resources/dicts/thoughts/alive/general.json

```
@@ -2210,16 +3801,22 @@
             "Is taken over by a coughing fit",
             "Is frustrated by {PRONOUN/m_c/poss} health",
             "Hisses at r_c for seemingly no reason",
-            "Has been experiencing changes in {PRONOUN/m_c/poss} behavior",
+            "Has been experiencing changes in {PRONOUN/m_c/poss} behaviour",
```

```suggestion
            "Has been experiencing changes in {PRONOUN/m_c/poss} behavior",
```
https://github.com/ClanGenOfficial/clangen/wiki/%5BWriting%5D-%E2%80%90-Basic grammar rules indicates american spelling of words